docs/operate/get-started/other-hardware/hello-world-module.md

```diff
@@ -240,7 +240,7 @@ You need to add some sensor-specific code to support the sensor component.
 
     ```json {class="line-numbers linkable-line-numbers" data-start="8"}
     {
-      "api": "rdk:component:sensor",
+      "api": "rdk:component:camera",
```

Comment:
?

Reply:
```suggestion
      "api": "rdk:component:sensor",
```
Should have caught this in the diff, sorry.